manifests/keyring.pp

```diff
@@ -59,6 +62,15 @@
         source  => $source,
         content => $content,
       }
+
+      if $dearmor {
+        exec { "dearmor-${name}":
+          command     => "gpg --dearmor ${file} && mv ${file}.gpg ${file}",
```

Comment:
I'm not 100% sure, but I think this would require me to do something like `['bash', '-c' "gpg --dearmor ${file} && mv ${file}.gpg ${file}"]` as I can't use `&&` with the array syntax. Are the benefits of the array notation worth it?